src/boot/ksdk1.1.0/warp-kl03-ksdk1.1-boot.c

```diff
@@ -1654,6 +1807,291 @@ main(void)
 				break;
 			}
 
+			/*
+			 *	Stress test, including bit-wise toggling and checksum, as well as extensive add and mul, with continous read and write to I2C MMA8451Q
+			 */
+			case 'y':
+			{
+				/*	
+				 *	Setup LED pins for FRDZ KL03
+				 */
+
+				enum
+				{
+					kRed	= GPIO_MAKE_PIN(HW_GPIOB, 10),
+					kGreen	= GPIO_MAKE_PIN(HW_GPIOB, 11),
+					kBlue	= GPIO_MAKE_PIN(HW_GPIOB, 13),
+				};
+
+				/*	
+				 *	I2C MMA8451Q initialization
+				 */
+
+				WarpStatus i2cWriteStatusA, i2cWriteStatusB;
+				WarpStatus i2cReadStatusX = kWarpStatusDeviceCommunicationFailed;
+				WarpStatus i2cReadStatusY = kWarpStatusDeviceCommunicationFailed;
+				WarpStatus i2cReadStatusZ = kWarpStatusDeviceCommunicationFailed;
+
+				enableSssupply(menuSupplyVoltage);
+				enableI2Cpins(1);
+
+				i2cWriteStatusA = writeSensorRegisterMMA8451Q(0x09 /* register address F_SETUP */,
+										0x00 /* payload: Disable FIFO */,
+										1);
+
+				i2cWriteStatusB = writeSensorRegisterMMA8451Q(0x2A /* register address */,
+										0x03 /* payload: Enable fast read 8bit, 800Hz, normal, active mode */,
+										1);
+
+				uint8_t accValue[3] = {0,0,0};
+
+				/*	
+				 *	Fill up the remaining memory space using a fixed size array
+				 *	The size of this array is highly dependent on the firmware code size
+				 */
+
+				int NUMBER = 200;

```

Comment:
Still magic numbers. These should be in a config enum as we've discussed a few times now.